pkg/monitor/datapath_policy.go

```diff
@@ -62,12 +61,23 @@ type PolicyVerdictNotify struct {
 	// data
 }
 
-// DecodePolicyVerdictNotify will decode 'data' into the provided PolicyVerdictNotify structure
-func DecodePolicyVerdictNotify(data []byte, pvn *PolicyVerdictNotify) error {
-	return pvn.decodePolicyVerdictNotify(data)
+// Dump prints the message according to the verbosity level specified
+func (pn *PolicyVerdictNotify) Dump(args *api.DumpArgs) {
+	pn.DumpInfo(args.Buf, args.Data, args.Format)
 }
 
-func (n *PolicyVerdictNotify) decodePolicyVerdictNotify(data []byte) error {
+// GetSrc retrieves the sorce endpoint for the message.
+func (n *PolicyVerdictNotify) GetSrc() uint16 {
+	return n.Source
+}
+
+// GetDst retrieves the destination endpoint for the message.
+func (n *PolicyVerdictNotify) GetDst() uint16 {
+	return uint16(n.RemoteLabel)
```

Comment:
I believe `RemoteLabel` is not an endpoint identity, but a security identity. Also IIUC for `POLICY_INGRESS` it is the source label, and in that case the src/dst nomenclature doesn't apply cleanly (see [here](https://github.com/cilium/cilium/blob/1a6fbe49a84450cf528247d212796d335b8e2be8/bpf/bpf_lxc.c#L1990-L1993) for example).

Reply:
Thank you, I didn't notice that! I adapted the comment, let me know :)